### PR TITLE
chore(docs):capitalization and grammar fixes

### PR DESCRIPTION
**Pull request by steph-chan**
## Description

Minor update for title/header/name capitalization, clarity, and pronouns

## Related Issues

Related to #18284

### CODE REVIEW

File: docs/docs/sourcing-from-buttercms.md [hunk 1]
```diff
@@ -981,7 +977,7 @@ See their [API reference](https://buttercms.com/docs/api/) for more information
 
 This was an example meant to help you understand how ButterCMS works with Gatsby. You're now able to:
 
-- Create a ButterCMS repository and setting it up together with the Gatsby plugin
+- Create a ButterCMS repository and set it up together with the Gatsby plugin
 - Query data from ButterCMS for single pages, multiple pages, blog posts, and custom content fields
 
 If you got stuck, you can compare your code to the [gatsby-starter-buttercms](https://github.com/ButterCMS/gatsby-starter-buttercms). To learn more about ButterCMS, check out their [blog](https://buttercms.com/blog/). Their latest updates can be found [here](https://buttercms.com/blog/category/new-to-butter/).
```
Comment (marcysutton) [inline docs/docs/sourcing-from-buttercms.md]:
```suggestion
If you got stuck, you can compare your code to the [gatsby-starter-buttercms](https://github.com/ButterCMS/gatsby-starter-buttercms). To learn more about ButterCMS, check out their [blog](https://buttercms.com/blog/). Their latest updates can be found on [buttercms.com](https://buttercms.com/blog/category/new-to-butter/).
```

**Comment by steph-chan** [comment]
Thanks for the feedback! I'll work on getting these fixed.

File: docs/docs/sourcing-from-buttercms.md [hunk 2]
```diff
@@ -121,11 +121,9 @@ Adding a CMS-powered page to your app involves three easy steps:
 1. Populate the content
 1. Integrate into your application
 
-If you need help after reading this, contact us via email or livechat.
-
 #### Create the page structure
 
-Create a new Page and define it's structure using our Page Builder. Create an example homepage.
+Create a new Page and define its structure using Page Builder. Let's create an example homepage.
```
Comment (marcysutton) [inline docs/docs/sourcing-from-buttercms.md]: "Let's" is introduced here but it doesn't follow the style guide. Is there another way you can phrase this?

Comment (marcysutton) [reply docs/docs/sourcing-from-buttercms.md]: I want to make sure this is merged in in time for Hacktoberfest, how about this: 
```suggestion
Create a new Page and define its structure using Page Builder. Create an example homepage to follow along with this guide:
```

**Comment by marcysutton** [comment]
I seem to have messed this PR up trying to merge master into it: I'm going to update and push directly to your branch so we can resolve conflicts and merge it!

**Pull request merged**